### ISSUE TITLE
Workflow does not toggle theme in Alfred 4

### ISSUE THREAD

**Issue by MrXcitement** [AlexMartinFR/mojave-alfred-accented-themes#2]
I recently updated to Alfred 4 and the theme 'toggle' workflow no longer works. I saw that the Application name is set to 'Alfred 3'. I was able to work around this issue by replacing the line:

```
alfredApp = "Alfred 3
```
with:

```
alfredAppName = "Alfred 3"
if (Application("Alfred 4").running()) {
	alfredAppName = "Alfred 4"
}
alfredApp = Application(alfredAppName)
```
I would have provided a pull-request, however the workflow is stored as a binary archive and I did not know how to provide an edit to just the plist file.

**Comment by alexmartinfr** [comment]
You wrote a fix before I even noticed this bug, nice!
I've updated the workflow in the repo, with proper versioning, releases notes and credits ;)

**Comment by MrXcitement** [comment]
My fix above was a naive solution that worked on my machine running Alfred 4. This is not the case, when testing with Alfred 3 installed, the workflow failed. I had incorrectly understood that the ```Application("AppName").running()``` check would return false if the app was not installed. It actually throws an error if the app is not installed, thus the workflow fails. I did a little more research and noticed you get an environment variable for the current theme background and noticed I could get the current Alfred version. I use the first character from the version as the major version (this should work for 5 more major versions) I have tested the below solution in both Alfred 4 and Alfred 3 running under macOS Mojave 10.14 and the solution works in both cases. To make the change, you will need to remove the previous solution and then after the line:
```
ObjC.import('stdlib')
```
add these lines to get the currently installed version of Alfred:
```
alfredMajorVer = $.getenv('alfred_version').charAt(0)
alfredAppName = "Alfred " + alfredMajorVer
alfredApp = Application(alfredAppName)
```
Sorry for the previous semi-working solution.
